managedApplication/DeployMoodleManagedApp.md

```diff
@@ -66,15 +66,20 @@ to manage we'll put these parameter values into environment variables.
 For convenience our `mainTemplate.json` file has defaults for all
 values. This means that there is no need to provide parameters in the
 commandline, though you can override the defaults if you want to by
-adding the `--parameters` attribute. This attribute can take either 
+adding the `--parameters` attribute. This attribute can take either
 a JSON string or a filename (preceded with an '@', e.g. '--parameters @parameters.json`) containing a JSON
 definition for the paramters, e.g.
 
-
-    {
-        "parameterName": "value",
-        "AnOtherParameter"" "another value"
+``` json
```

Comment:
These scripts are written to be executed in SimDem. In SimDem code blocks that are denoted by backticks are executable. Therefore this block should be formatted with indents (which is valid md).

Reply:
Got it. Removed.